build.gradle

```diff
@@ -4,7 +4,7 @@ import java.net.http.HttpResponse
 import java.nio.file.Paths
 
 plugins {
-    id "org.jetbrains.intellij" version "1.16.0"
+    id "org.jetbrains.intellij" version "1.17.0"
```

Comment:
this is an experiment for something, will remove before merge

Reply:
reminder to remove